sphinxcontrib/spelling/__init__.py

```diff
@@ -43,4 +55,7 @@ def setup(app):
     app.add_config_value('spelling_ignore_importable_modules', True, 'env')
     # Add any user-defined filter classes
     app.add_config_value('spelling_filters', [], 'env')
-    return {"parallel_read_safe": False}
+    return {
+        "parallel_read_safe": True,
```

Comment:
Is it also write-safe?

Reply:
It could be made to be, but it probably isn't since it logs all output to a single file for review.

Reply:
@dhellmann I must warn you that **it is `True` by default**: https://www.sphinx-doc.org/en/master/extdev/index.html#extension-metadata.

So by not specifying it you basically announce that it's write-safe.

Reply:
Done